pkg/cache/bigquerycache/bigquery.go

```diff
@@ -0,0 +1,223 @@
+package bigquerycache
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"cloud.google.com/go/bigquery"
+	uuid2 "github.com/google/uuid"
+	"github.com/openshift/sippy/pkg/apis/cache"
+	sippybq "github.com/openshift/sippy/pkg/bigquery"
+	"github.com/openshift/sippy/pkg/cache/compressed"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/api/iterator"
+)
+
+// https://cloud.google.com/bigquery/quotas#streaming_inserts
+// Maximum row size 	10 MB 	Exceeding this value causes invalid errors.
+// HTTP request size limit 	10 MB Exceeding this value causes invalid errors.
+const (
+	cachedTable     = "cached_data"
+	partitionColumn = "modified_time"
+	chunkSize       = 7000000 // ~7MB to stay under the max row limit
+)
+
+type Cache struct {
```

Comment:
Godoc would be good here, what this is for, why we did it, how it interacts with the warm redis cache.